host/src/components/HeaderGame.jsx

```diff
@@ -1,14 +1,18 @@
 import React from "react";
```

Comment:
Design team has updated the pagination component so these adjustments, and those in Pagination.jsx, handle that.